desci-server/tsconfig.json

```diff
@@ -29,7 +30,9 @@
     "paths": {
       // Overrides a built-in Response type
       "express": ["./src/types/express"]
-    }
+    },
+    // "strictNullChecks": true,
+    // "exactOptionalPropertyTypes": true
```

Comment:
Too many fixes? :grin:

Reply:
helped to identify potential bugs, we should probably keep it on sometimes in the future

Reply:
Yup, should strictify the heck outta this project, but it takes some initial work :)